invenio_app_ils/cli.py

```diff
@@ -1242,6 +1243,39 @@ def create_userprofile_for(email, username, full_name):
         db.session.commit()
 
 
+@click.group()
+def fixtures():
+    """Create initial data and demo records."""
+
+
+@fixtures.command()
+@with_appcontext
+def pages():
+    """Register CDS static pages."""
+    def page_data(page):
+        return pkg_resources.resource_stream(
+            'invenio_app_ils', os.path.join('templates/static_pages', page)
+        ).read().decode('utf8')
+
+    pages = [
+        Page(url='/about',
+             title='About',
+             description='About',
+             content=page_data('about.html'),
+             template_name='invenio_pages/default.html'),
+        Page(url='/contact',
+             title='Contact',
+             description='Contact',
+             content=page_data('contact.html'),
+             template_name='invenio_pages/default.html'),
+    ]
+    with db.session.begin_nested():
+        Page.query.delete()
+        db.session.add_all(pages)
+    db.session.commit()
+    click.echo('static pages created:)')
```

Comment:
```suggestion
    click.echo('static pages created :)')
```
and if you use Black will probably change all your quotes `'` -> `"`